R/npn_data_download.R

```diff
@@ -910,9 +911,15 @@ npn_get_data <- function(
   # assume that memory could be a limitation and wrangle data 5000 rows at a
   # time and append to the CSV file specified in `download_path`
   if (is.null(download_path)) {
-    dtm <-
-      httr2::resp_body_json(resp, simplifyVector = TRUE) %>%
-      wrangle_dl_data()
+    dtm <- httr2::resp_body_json(resp, simplifyVector = TRUE)
+    # If dtm is an empty list because there are no records, set equal to NULL
+    # and print message
+    if (is.null(dim(dtm))) {
+      dtm <- NULL
```

Comment:
It might be better to return an empty tibble with `dplyr::tibble()` rather than `NULL` in terms of possibly breaking things downstream (although `bind_rows(NULL, data)` works fine).